File: Simon_Visual_Basic/PruebaLabels 16/PruebaLabels/Inicio.vb
Module Inicio

    '  Public _archivoPuntajes As String = "C:\ListaPuntajes.xml"

    Public _archivoPuntajes As String = Application.StartupPath & "\ListaPuntajes.xml"



    Sub Main()

        Try
            Dim frm As New FrmSimon

            Application.Run(frm)


        Catch ex As Exception

            MessageBox.Show(ex.Message)

        End Try



    End Sub





End Module

File: Tetris_C#/t2/Inicio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication1
{
    public class Inicio
    {
        public static string Ruta = Application.StartupPath + "\\Tetris.xml";


        public static void Main(string[] vector)
        {
            try
            {
                Application.Run(new FrmPrincipal());

            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message.ToString());
            }


        }

    }
}
